src/components/Tasks/Toolbar.tsx

```diff
@@ -0,0 +1,67 @@
+import * as React from 'react';
+
+import * as EteSync from 'etesync';
+
+import Switch from '@material-ui/core/Switch';
+import FormControlLabel from '@material-ui/core/FormControlLabel';
+import IconButton from '@material-ui/core/IconButton';
+import MoreVertIcon from '@material-ui/icons/MoreVert';
+import Menu from '@material-ui/core/Menu';
+import MenuItem from '@material-ui/core/MenuItem';
+
+import QuickAdd from './QuickAdd';
+
+import { PimType } from '../../pim-types';
+
+interface PropsType {
+  defaultCollection: EteSync.CollectionInfo;
+  onItemSave: (item: PimType, journalUid: string, originalItem?: PimType) => Promise<void>;
+  showCompleted: boolean;
+  setShowCompleted: React.Dispatch<React.SetStateAction<boolean>>;
+}
+
+export default function Toolbar(props: PropsType) {
+  const { defaultCollection, onItemSave, showCompleted, setShowCompleted } = props;
+
+  const [anchorEl, setAnchorEl] = React.useState<null | HTMLElement>(null);
+
+  const handleClick = (event: React.MouseEvent<HTMLElement>) => {
+    setAnchorEl(event.currentTarget);
+  };
+
+  const handleClose = () => {
+    setAnchorEl(null);
+  };
+
+  return (
+    <div style={{ display: 'flex', justifyContent: 'space-between' }}>
+      {defaultCollection && <QuickAdd onSubmit={onItemSave} defaultCollection={defaultCollection} />}
+
+      <div>
+        <IconButton
+          aria-label="more"
+          aria-controls="long-menu"
+          aria-haspopup="true"
+          onClick={handleClick}
+        >
+          <MoreVertIcon />
+        </IconButton>
+        <Menu
+          id="simple-menu"
+          anchorEl={anchorEl}
+          keepMounted
+          open={Boolean(anchorEl)}
+          onClose={handleClose}
+        >
+          <MenuItem>
+            <FormControlLabel
+              label="Show completed"
+              control={<Switch checked={showCompleted} onChange={(_e, checked) => setShowCompleted(checked)} />}
```

Comment:
I think that if it's a switch, the switch should be on the left and the label on the right, and if it's checkbox the way it is now. What do you think? It just looks odd to me like this.

Reply:
Do you mean the other way around? I think the switch is currently on the left

Reply:
Yeah, the other way around. Sorry, I meant switch on the right. :)

Reply:
I put it that way and you're right, looks much better.

Reply:
Great. :)